Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) Greenbone Networks GmbH
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# Build and development command for the scan management backend project.
+# important commands are documented and machine-runnable at the same time here.
+
+PROJECT := go-project-template
+GOPATH := $(shell go env GOPATH)
+
+.PHONY: help
+# Show this help
+help: # So I managed to get the help printing with the commment line above it
+	@awk '/^#/{c=substr($$0,3);next}c&&/^[[:alpha:]][[:alnum:]_-]+:/{print "\033[36m", substr($$1,1,index($$1,":")), "\033[0m", c}1{c=0}' $(MAKEFILE_LIST) | column -s: -t
+
+.PHONY: project-setup
+# Setup project for development. Install project dependencies, etc.
+project-setup: debian-os-package-install python-package-install go-package-install
+
+.PHONY: debian-os-package-install
+# Install debian apt packages (requires sudo)
+debian-os-package-install:
+	sudo apt update
+	sudo apt install golang golang-1.18-go
+
+# IF not required remove
+.PHONY: go-package-install
+# Install required go binaries
+go-package-install:
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin v1.50.1
```

Comment:
I wonder if we want the version be included here... but I have no clear opinion if or if not... any arguments for setting a fixed the version here or the other way just install latest always?